### PR TITLE
Updated Tenting101

### PR DESCRIPTION
**Pull request by rikkikendall**
The overall purpose of this PR is for the Tenting101 section at the top of the login page. 

## Description

Created stepper for Tenting101. Routed the page from the login page and back. Utilizing a tenting101 container that imports multiple components. 
Involves gathering information from Policy Changes to include the Line Monitor Tent Registration
and Tenting Dates/Information.

## Motivation and Context

Enhancement for user experience Tenting in Kville. Provides a step by step resource on how to tent, if you don't know how to tent then you won't be able to use our app. 

## How Has This Been Tested?

Test
How to test:
Run yarn dev:start
Try the Tenting101 feature at the top of the login page.
More info for testing coming...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

**Comment by amanmibra** [comment]
Hey, before I do the review, I notices your commit messages, and I thought I would share this with you:
https://github.com/agis/git-style-guide#commits

This goes over basic conventions of many git practices, including commits and commit messages. Read it and follow it and it'll be very helpful in the future going forward.

**Comment by amanmibra** [comment]
Also, did the PR template not show up for you when you made the PR? Please use it for any PRs from now on: 

Go ahead and update your message above with the following (just copy and paste):
```
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
```

File: app/javascript/src/containers/Login.jsx [hunk 1]
```diff
@@ -50,20 +62,28 @@ class Login extends Component {
             logoutUser,
             signup,
             signupNewTeam,
-            clearError,
```
Comment (amanmibra) [inline app/javascript/src/containers/Login.jsx:53]: You removed an important redux action, clearError, during your merging, please add it back. When dealing with merge conflicts from staging, if it does not affect the feature you are working on, always keep the new code.

File: app/javascript/src/containers/Login.jsx [hunk 2]
```diff
@@ -138,7 +159,7 @@ const mapDispatchToProps = (dispatch) => {
       logoutUser: logout,
       signup: signup,
       signupNewTeam: signupNewTeam,
-      clearError: clearError,
```
Comment (amanmibra) [inline app/javascript/src/containers/Login.jsx]: Same thing, add this line back.

File: app/javascript/src/containers/Login.jsx (hunk 1 above)
Comment (amanmibra) [reply app/javascript/src/containers/Login.jsx:53]: Just add the same line back

File: app/javascript/src/containers/Tenting101.jsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeStep: 0,
+      activeItem: 'tenting101',
+    };
+    this.handleReset = this.handleReset.bind(this);
+  }
+  handleClick = (e, data) => {
+    console.log('data', data);
```
Comment (amanmibra) [inline app/javascript/src/containers/Tenting101.jsx]: Remove console.logs. Cannot have in production.

File: app/javascript/src/containers/Tenting101.jsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+            </Menu.Item>
+            <Menu.Item
+             id='login'
+             active={path === '/login'}
+             onClick={this.handleClick}>
+             <Icon name="sign in" />
+             Login
+             </Menu.Item>
+          </ Menu>
+          </ div>
```
Comment (amanmibra) [inline app/javascript/src/containers/Tenting101.jsx]: Lines 95-96
Can you remove that extra spacing between `</` and `Menu>`. Same thing for the `div` below it.

File: app/javascript/src/components/tenting/Step1.jsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+// import FAQ page
+import FAQ from './FAQ';
+
+class Step1 extends Component {
+  render () {
+    const square = { width: 175, height: 175 };
+    return (
+      <div className="body">
+       <Image centered src={kvilleLogo} size='massive'/>
+       <Message centered header='Step 1' size='large' content='Find your tent members and establish a Tent Captain.'/>
```
Comment (amanmibra) [inline app/javascript/src/components/tenting/Step1.jsx]: `Message` does not have a centered prop. Please check docs to confirm props before using them as they throw errors or warnings (check your console, and you can see them).

**Review by amanmibra — CHANGES_REQUESTED** [review]
Looks good, just make the changes that I request, and also update your NavBar for the page. Once you do that just @ me and then you will prob be good to go! :+1:

**Comment by amanmibra** [comment]
@rikkikendall Also, open your console and see the warnings your code is setting off. Work on getting rid of those as well.

**Comment by rikkikendall** [comment]
Sounds great, should have implemented those changes. Now need to decide on design changes for NavBar on the top, whether that should be like the login with the white background or the homepage with the inverted blue.

**Review by amanmibra — APPROVED** [review]

**Comment by amanmibra** [comment]
Good work @rikkikendall I have approved your PR, but please remember to always pull and merge staging by doing the following:

```
$ git pull
$ git pull origin staging
```

**Pull request merged**